cmd/buildah/tag.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/projectatomic/buildah/util"
+	"github.com/urfave/cli"
+)
+
+var (
+	tagDescription = "Adds one or more additional names to locally-stored image"
+	tagCommand     = cli.Command{
+		Name:        "tag",
+		Usage:       "Add an additional name to a local image",
+		Description: tagDescription,
+		Action:      tagCmd,
+		ArgsUsage:   "IMAGE-NAME [IMAGE-NAME ...]",
+	}
+)
+
+func tagCmd(c *cli.Context) error {
+	args := c.Args()
+	if len(args) == 0 {
```

Comment:
Don't you need to add both an image and a name or names here?  I thought you'd want to check len(args) > 1 here.